NEW: Add $object to 'printFieldListValue' hookmanager

milenmk opened the pull request:
NEW: Adding this option give the ability to pass commande options as object for hooks (like $object->id etc.). Otherwise it is not possible.

milenmk committed:
Add $object to 'printFieldListValue' hookmanager

Adding this option give the ability to pass commande options as object for hooks (like $object->id etc.). Otherwise it is not possible.

stickler-ci committed:
Fixing style errors.

eldy commented on htdocs/commande/list.php:
```diff
@@ -1779,7 +1779,8 @@
 		$parameters = array('arrayfields'=>$arrayfields, 'obj'=>$obj, 'i'=>$i, 'totalarray'=>&$totalarray);
-		$reshook = $hookmanager->executeHooks('printFieldListValue', $parameters); // Note that $action and $object may have been modified by hook
+		$object = $generic_commande;
+		$reshook = $hookmanager->executeHooks('printFieldListValue', $parameters, $object); // Note that $action and $object may have been modified by hook
```
Here $object is not a complete object. Providing it as $object parameter let think it is. All data of $object = $generic_commande should already be inside $obj, shouldn't it ?

milenmk committed:
Merge branch 'Dolibarr:develop' into develop

milenmk committed:
PSR-12 compliance

Fixed "Unnecessary double quotes" (replaced with single quotes)
Fixed "Traditional array used" (converted to short syntax)
Fixed some invertedIF's
Fixed lines longer than 120 columns
Fixed some methods "Cyclomatic complexity" by braking them to few methods
Fixed some obsolete attributes by removing them
Fixed some grammar
Fixed nested IF's by merging them
Fixed shor php tags used ("<?=" instead of "<?php")

Added parameter's type declaration
Added propertie's type declaration
Added return type declaration for methods
Added "declare(strict_types=1);" to files

Removed redundant optional arguments

Strict comparissons applied ("===" instead of "==" and "!==" instead of "!=")

stickler-ci committed:
Fixing style errors.

milenmk committed:
minor

milenmk committed:
Merge branch 'develop' of https://github.com/milenmk/dolibarr into develop

milenmk committed:
Merge branch 'Dolibarr:develop' into develop
